### PR TITLE
Created grades.java

### LINKED ISSUE #267: I want to add Java Program to demonstrate the use of If else-if ladder using grading System
implement Java Program to demonstrate the use of If else-if ladder using grading System

**Assigned to:**

- [x] java: **@kn0wabhi**

### PR DESCRIPTION
**Pull request by kn0wabhi**
**java Program to demonstrate the use of If else-if ladder.
 program of grading system** 
- for fail (&lt;50),
• D grade -marks&gt;=50 &amp;&amp; marks&lt;60
• C grade- marks&gt;=60 &amp;&amp; marks&lt;70,
• B grade- marks&gt;=70 &amp;&amp; marks&lt;80 ,
• A grade- marks&gt;=80 &amp;&amp; marks&lt;90
• and A+ - marks&gt;=90 &amp;&amp; marks&lt;100.

Fixes #267 

## Type of change

-   [x] New feature (non-breaking change which adds functionality)

# Checklist:

-   [x] My code follows the style guidelines of this project
-   [x] I have performed a self-review of my own code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [x] I have made corresponding changes to the documentation
-   [x] My changes generates no new warnings
-   [x] I have added tests that prove my fix is effective or that my feature works
-   [x] New and existing unit tests pass locally with my changes
-   [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by kaustubhgupta** [comment]
@kn0wabhi   do the following things:
- Raise an issue which this PR solves (Fill that template carefully)
- Wait for it to be assigned to you
- Fill this PR template accordingly

**Comment by kn0wabhi** [comment]
Code Description: 
Java Program to demonstrate the use of If else-if ladder.
• program of grading system for fail (<50),
• D grade -marks>=50 && marks<60
• C grade- marks>=60 && marks<70,
• B grade- marks>=70 && marks<80 ,
• A grade- marks>=80 && marks<90
• and A+ - marks>=90 && marks<100.


fixes : #267

Type of change
 # New feature (non-breaking change which adds functionality)

Checklist:
 # My code follows the style guidelines of this project
 # I have performed a self-review of my own code
 # I have commented my code, particularly in hard-to-understand areas
 # I have made corresponding changes to the documentation
 # My changes generates no new warnings
 # I have added tests that prove my fix is effective or that my feature works
 # New and existing unit tests pass locally with my changes
 # Any dependent changes have been merged and published in downstream modules

**Comment by kn0wabhi** [comment]
Code Description:
Java Program to demonstrate the use of If else-if ladder.
• program of grading system for fail (<50),
• D grade -marks>=50 && marks<60
• C grade- marks>=60 && marks<70,
• B grade- marks>=70 && marks<80 ,
• A grade- marks>=80 && marks<90
• and A+ - marks>=90 && marks<100.

fixes : #267

Type of change
-New feature (non-breaking change which adds functionality)


Checklist:

-My code follows the style guidelines of this project
-I have performed a self-review of my own code
-I have commented my code, particularly in hard-to-understand areas
-I have made corresponding changes to the documentation
-My changes generates no new warnings
-I have added tests that prove my fix is effective or that my feature works
-New and existing unit tests pass locally with my changes
-Any dependent changes have been merged and published in downstream modules

**Comment by kaustubhgupta** [comment]
@kn0wabhi Edit the PR template instead of posting it in comment

**Comment by kn0wabhi** [comment]
**@kaustubhgupta**  As per you asked i have edited PR and issue templates

File: Java/grades.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
+/*Java Program to demonstrate the use of If else-if ladder.
```
Comment (kaustubhgupta) [inline Java/grades.java]: Remove this doc string

File: Java/grades.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+• program of grading system for fail (&lt;50),
+• D grade -marks&gt;=50 &amp;&amp; marks&lt;60
+• C grade- marks&gt;=60 &amp;&amp; marks&lt;70,
+• B grade- marks&gt;=70 &amp;&amp; marks&lt;80 ,
+• A grade- marks&gt;=80 &amp;&amp; marks&lt;90
+• and A+ - marks&gt;=90 &amp;&amp; marks&lt;100.*/
+import java.util.Scanner;
+public class Grade
```
Comment (kaustubhgupta) [inline Java/grades.java]: Class name should be same as file name

File: Java/grades.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+{
+public static void main(String args[])
+{
+System.out.println(&quot;Enter Marks: &quot;);
+ABC S=new ABC();
+S.display();
```
Comment (kaustubhgupta) [inline Java/grades.java]: - Reformat the whole code with appropriate spaces
- comments which you wrote, in the beginning, should be placed at appropriate grade assigned

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

File: Java/grades.java (hunk 1 above)
Comment (kn0wabhi) [reply Java/grades.java]: done

File: Java/grades.java (hunk 2 above)
Comment (kn0wabhi) [reply Java/grades.java]: done

File: Java/grades.java (hunk 3 above)
Comment (kn0wabhi) [reply Java/grades.java]: done

**Review by kn0wabhi — COMMENTED** [review]
**@kaustubhgupta**  I have made all the changes that you have requested.

File: Java/grades.java [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+{
+   Scanner A = new Scanner(System.in);
+   int marks=A.nextInt();
+   public void display()
+{
+if(marks&lt;50)                                   /* fail (&lt;50)*/
+{
+    System.out.println(&quot;Fail&quot;);
+}
+else if(marks&gt;=50 &amp;&amp; marks&lt;60)      /* D grade -marks&gt;=50 &amp;&amp; marks&lt;60*/
```
Comment (kaustubhgupta) [inline Java/grades.java:21]: &amp ? Fix this

File: Java/grades.java [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+{
+    System.out.println(&quot;D Grade&quot;);
+}
```
Comment (kaustubhgupta) [inline Java/grades.java:24]: The program is still not formated properly, give spaces and indents

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Comment by YashSaxena9** [comment]
The presence of web-based symbol codes indicates a suspicion of plagiarism, PR is being rejected for the same reason.

**Pull request closed without merging**